ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -104,8 +157,18 @@ function get_hawq_password() {
   done
 }
 
+function sync_rps_configuration() {
+  if [[ -d $GPHOME/ranger/etc/ && ! -z $HAWQ_STANDBY_HOST ]]; then
```

Comment:
I'd print a statement telling user what the script is about to do as well as the status once done